src/modules/dashboard/dashboard.hooks.ts

```diff
@@ -14,7 +15,11 @@ export const useGetRequestsList = (queryParams: IFetchComplaintListRequest) => {
     created_by_me:queryParams?.created_by_me===true ? true:undefined,
   }
   const { data, isLoading, isFetching } = useQuery(['complaint-list',params], () =>
-    getRequestList(path,params)
+    getRequestList(path,params),{
+      onError:(e)=>{
+        toast.error("unable to fetch")
```

Comment:
Can we have specific message here like Failed to fetch Request list with e.message